### ISSUE TITLE
TypeError: '<' not supported between instances of 'NoneType' and 'float

### ISSUE THREAD

**Issue by peterfarrell** [dalibo/pg_activity#168]
Upon launching on Python 3.8.6 using this package version 2.0.2, the follow error is generated:

```
  File "/usr/local/bin/pg_activity", line 296, in <module>
    ui.main(term, dataobj, host, options, dsn)
  File "/usr/local/lib/python3.8/site-packages/pgactivity/ui.py", line 215, in main
    views.screen(
  File "/usr/local/lib/python3.8/site-packages/pgactivity/views.py", line 501, in screen
    processes.set_items(sorted_processes(processes, key=ui.sort_key, reverse=True))
  File "/usr/local/lib/python3.8/site-packages/pgactivity/activities.py", line 194, in sorted
    return builtins.sorted(
TypeError: '<' not supported between instances of 'NoneType' and 'float
```

**Comment by dlax** [comment]
Thanks for the report. In order to investigate, I'll need more information:

* what postgresql version are you using?
* are you running pg_activity using the "postgres" user (or the user running the postgresql server)?
* can you share the command-line and arguments used to launch pg_activity?

**Comment by dlax** [comment]
I released a version 2.0.3 fixing (hopefully) this issue; it's available on PyPI. Thanks for the report.

**Comment by peterfarrell** [comment]
@dlax that seems to have fixed the issue. Thanks!
